terraform/modules/benchmark/variable.tf

```
@@ -62,3 +62,21 @@ variable "project" {
   type        = string
   default     = "Forest-DEV" # Alternative: "Default"
 }
+
+variable "NEW_RELIC_API_KEY" {
+  description = "New Relic API KEY"
+  default     = ""
+  type        = string
+}
+
+variable "NEW_RELIC_ACCOUNT_ID" {
+  description = "New Relic Account ID"
+  default     = ""
+  type        = string
+}
+
+variable "NEW_RELIC_REGION" {
+  description = "The New Relic Plathom Region"
```

https://www.urbandictionary.com/define.php?term=Plathom
```suggestion
  description = "The New Relic Platform Region"
```